src/storage-proofs/block.ts

```diff
@@ -0,0 +1,36 @@
+import Common, { Chain, Hardfork } from '@ethereumjs/common';
+import { bufferToHex } from 'ethereumjs-util';
+import blockFromRpc from '@ethereumjs/block/dist/from-rpc';
```

Comment:
What is this for?

Reply:
This serializes a block in the correct RLP encoding.